File: FirstTask/Python/Insertionsort/insertionsortMain.py
import random
from insertionsortAlgo import *
def sortRandomArray():
    randomArray = [random.randint(1,100) for i in range(10)]
    print("Array to sort: ", randomArray)
    sortedArray = insertionsort(randomArray)
    print("Array after sorting: ",sortedArray)

sortRandomArray()

File: FirstTask/JavaScript/Insertionsort/insertionsortMain.js
import {insertionsort} from "./insertionsortAlgo.js"
const array = [];
for (let i = 0; i < 10; i++) {
    array.push(Math.floor(Math.random() * 100));
}
console.log("Array to sort: ", array);
let sortedarray = insertionsort(array);
console.log("Sorted Array: ", sortedarray)

insertionsort()